.github/workflows/build.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       runs-on: ${{ matrix.os }}
       strategy:
         matrix:
-          os: [ubuntu-20.04]
+          os: [ubuntu-20.04, ubuntu-22.04, ubuntu-24.04]
```

Comment:
Do we intend to upgrade Heroku to 24.04 soon, and if so, does it require an incremental upgrade to 22.04 first? If not, we can probably skip 22.04 entirely and save the cycles. Otherwise LGTM.

Reply:
*shrug* don't know how it will be until we do it. For testing purposes, I'm fine either way: since this is a public repo, we don't pay for the action runs so it doesn't really affect us :smile:

Reply:
Understood—let's run all three versions for a bit until we know more, but I prefer not to waste cycles if we don't have to (even if they are 'free').